client/ayon_maya/plugins/publish/extract_layout.py

```diff
@@ -168,6 +172,22 @@ def process(self, instance):
                     "y": local_rotation[1],
                     "z": local_rotation[2]
                 }
+                if allow_obj_transforms:
+                    child_transforms = cmds.listRelatives(
```

Comment:
You seem to have reverted this to `listRelatives` for some reason. This is wrong.
- `get_all_children` already gets the child relatives.
- Also, you're lacking the `fullPath=True`.  But it should be a `cmds.ls()` call with `long=True` actually - which is what it was before. Not sure why it was reverted?

Reply:
You can test it by your own. If it is cmds.ls() by the case, it would mess up the entire transformation data and once you load from the layout loader, it won't replicate the same scene as your workfile has been exported.

Reply:
Here's an example:
<img width="728" height="137" alt="image" src="https://github.com/user-attachments/assets/5d40a069-0186-4ca0-965e-41ae4a263c91" />

<img width="715" height="149" alt="image" src="https://github.com/user-attachments/assets/d21ef0f9-ebd0-4ac7-946b-f42f15408be3" />